hedera-node/src/main/java/com/hedera/services/txns/token/operation/TokenBurnLogic.java

```diff
@@ -0,0 +1,36 @@
+package com.hedera.services.txns.token.operation;
+
+import com.hedera.services.state.enums.TokenType;
+import com.hedera.services.store.AccountStore;
+import com.hedera.services.store.TypedTokenStore;
+import com.hedera.services.store.models.Id;
+import com.hedera.services.store.models.OwnershipTracker;
+
+import java.util.List;
+
+public class TokenBurnLogic {
+
+	public void burn(
```

Comment:
`tokenStore` and `accountsStore` should be part of the properties in the `TokenBurnLogic` and be passed in the constructor